### DISCUSSION TITLE
When I back to students list just remain at original location

### DISCUSSION THREAD

**Discussion by JYLin915** [lonqie/SchaleDB#89] [Ideas and Suggestions]
Sometimes I just read information about some students to compare their skills. But it always goes back to the top of the list when I click the students' list. (I am not very good at English,  please bear with me.)

**Comment by lonqie** [comment]
The scroll position is preserved if you navigate backwards to the list page using the browser back/forward buttons.
If you instead enter the list page by the navigation bar or the student page it will return to the top as it is a new navigation.
